### ISSUE TITLE
Error in training model in R

### ISSUE THREAD

**Issue by thanish** [catboost/catboost#286]
Problem: {Error in catboost.get_feature_importance(model, learn_pool) : 
  atboost/library/json/writer/json_value.cpp:446: Not an unsigned integer}
catboost version: {0.6.3}
Operating System: {Windows}
CPU: {intel}
# GPU: {Nvidia}

I'm trying catboost for the 1st time. I get the above error after the model runs for the given iteration. 
I ran the code which is explained in the example 
https://tech.yandex.com/catboost/doc/dg/concepts/r-usages-examples-docpage/

Just pasting again the code I ran

train_path = system.file("extdata", 
                         "adult_train.1000", 
                         package="catboost")
test_path = system.file("extdata", 
                        "adult_test.1000", 
                        package="catboost")

column_description_vector = rep('numeric', 15)
cat_features <- c(3, 5, 7, 8, 9, 10, 11, 15)
for (i in cat_features)
  column_description_vector[i] <- 'factor'
train <- read.table(train_path, 
                    head = F, 
                    sep = "\t", 
                    colClasses = column_description_vector, 
                    na.strings='NAN')
test <- read.table(test_path, 
                   head = F, 
                   sep = "\t", 
                   colClasses = column_description_vector, 
                   na.strings='NAN')
target <- c(1)
train_pool <- catboost.load_pool(data=train[,-target], 
                                 label = train[,target])
test_pool <- catboost.load_pool(data=test[,-target], 
                                label = test[,target])
fit_params <- list(iterations = 100,
                   thread_count = 10,
                   loss_function = 'Logloss:Border=0.5',
                   ignored_features = c(4,9),
                   border_count = 32,
                   depth = 5,
                   learning_rate = 0.03,
                   l2_leaf_reg = 3.5,
                   train_dir = 'train_dir')
model <- catboost.train(pool, test_pool, fit_params)

**Comment by thanish** [comment]
Hi I have waited a while to get an answer for this. It was a lot of trouble trying to install catboost for R and when I was finally successful now I'm not able to run a model.@annaveronika can you please have a look at my issue.

**Comment by nikitxskv** [comment]
@thanish Hello!
I couldn't reproduce your error.
Could you update CatBoost and try again?

**Comment by thanish** [comment]
Hi @nikitxskv, please find in the below screenshots the code that I ran and the error It got. I believe I'm in the latest version of catboost 0.6.3.

I'm trying to run the code explained in the tutorial 
https://tech.yandex.com/catboost/doc/dg/concepts/r-quickstart-docpage/ 

![untitled](https://user-images.githubusercontent.com/4056145/38612648-03dc5ed6-3da5-11e8-8b92-40c069f7e715.png)

**Comment by nikitxskv** [comment]
@thanish There is no mistake.
At the moment, the newest version is 0.7.2
Please try to reinstall CatBoost by running the following commands:
`remove.packages('catboost')
devtools::install_url('https://github.com/catboost/catboost/releases/download/v0.7.2/catboost-`

**Comment by thanish** [comment]
Hey @nikitxskv I still get the same error after updating to the latest version of catboost

![image](https://user-images.githubusercontent.com/4056145/38655205-099c2fc2-3e31-11e8-9f6e-1a66da25b81c.png)

**Comment by LeData** [comment]
I had the same problem when trying to pass parameters as a dictionary in Python. It disappears when passing them 'manually'.

**Comment by nikitxskv** [comment]
@thanish @LeData Hi!
Could you upgrade CatBoost to the latest version(8.0) and try again?
We added a more detailed error message.

**Comment by annaveronika** [comment]
Closing because of inactivity. I hope everything works fine now, please feel free to reopen, if there are still issues.

**Comment by psygoh** [comment]
I am getting the exact same error message with v0.9.1.1 as well.

Error in catboost.get_feature_importance(model, learn_pool) : 
  atboost/library/json/writer/json_value.cpp:446: Not an unsigned integer
